frontend/socialapp/my-pov-app/app/services/auth/AuthService.tsx

```diff
@@ -0,0 +1,102 @@
+import { saveData, deleteData } from "app/utils/storage/securestore"
+
+const API_URL = "http://10.0.2.2:8000/users" // Backend API URL
```

Comment:
dont commit this

Reply:
removed url from component and created api component